wgpu/gui/base.py

```diff
@@ -301,17 +301,19 @@ def handle_event(self, event):
                 callback(event)
 
     def add_event_handler(self, *args):
-        """Register an event handler.
+        """Register an event handler to receice events.
```

Comment:
Typo :)

Reply:
I find the idea of running a spellchecker on CI increasingly interesting 🤔

Reply:
Yeah me too 😅